### PR TITLE
feat: IDKit v1 migration guide

### PR DESCRIPTION
**Pull request by 0xPenryn**
adds a migration guide and updates the announcement banner on the homepage.

several links may appear broken until other PR's are merged into idkit-v1-updates.

TODO: include links to blog posts once available.

### CODE REVIEW

**Review by pdtfh — CHANGES_REQUESTED** [review]

File: src/pages/reference/world-id-2-migration-guide.mdx [hunk 1]
```diff
@@ -0,0 +1,131 @@
+# World ID 2.0 Migration Guide
+
+This guide will help you migrate your application to use World ID 2.0, including upgrading from the beta version of IDKit (v0.5.1 or earlier) to the new release of IDKit (v1.0.0 or later). 
+We recommend you read the [World ID 2.0 Announcement](/world-id-2) to understand the new features and changes.
+
+<Note>
+    The [Breaking Changes](#breaking-changes) section outlines changes that must be addressed in order to successfully migrate your application to IDKit v1.
+    The [New Features](#new-features) and [Deprecated Functionality](#deprecated-functionality) sections outline changes that do not require immediate action.
+</Note>
+
```
Comment (pdtfh) [inline src/pages/reference/world-id-2-migration-guide.mdx:10]: We should start with the overall changes of World ID 2.0 & then talk about breaking changes in IDKit

File: src/pages/reference/world-id-2-migration-guide.mdx [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+**Intended only for use within another IDKit package**, and not intended to be used directly by applications.
+
+### `idkit-js` Monorepo
+
+The [GitHub repository for IDKit](https://github.com/worldcoin/idkit-js) has been updated to a monorepo, which contains the source code for the `@worldcoin/idkit-core`, `@worldcoin/idkit`, and `@worldcoin/idkit-standalone` packages.
+This refactor allows for easier maintenance and development of IDKit, and allows for much simpler creation of new IDKit packages for various frontend frameworks.
+
+<Note>Want support for a new frontend framework? Let us know by [opening an issue](https://github.com/worldcoin/idkit-js/issues) or build it and [submit a pull request](https://github.com/worldcoin/idkit-js/pulls)!</Note>
+
+### Email Sign In for Developer Portal
```
Comment (pdtfh) [inline src/pages/reference/world-id-2-migration-guide.mdx:109]:
```suggestion
### Use your email to sign in to the Developer Portal
```
or something along those lines. after coming back the title is confusing

File: src/pages/reference/v1-migration-guide.mdx [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+If your application uses vanilla JavaScript, you will need to install the new `@worldcoin/idkit-standalone` package instead of `@worldcoin/idkit`. 
+This package acts as a wrapper around the `@worldcoin/idkit` package for applications that do not use React.
+
+<Note>See the [IDKit Standalone](/reference/idkit#idkit-standalone) reference documentation for integration details.</Note>
+
+## New Features
+
+This section outlines new features introduced in IDKit v1 and other developer tools. These features are optional, and you can safely upgrade to IDKit v1 without making any changes described in this section.
+
+### Wallet Bridge
```
[the remark this answers is not in the document]
Comment (0xPenryn) [reply src/pages/reference/v1-migration-guide.mdx]: highlighted protocol internals as first reference and updated name

File: src/pages/reference/v1-migration-guide.mdx [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+WalletConnect was previously used to pass messages between IDKit and a user's identity wallet. This has been replaced with the new Wallet Bridge, which is a lightweight, open-source, and secure message passing service.
+
+If you previously set `walletConnectProjectId` in IDKit's parameters, you should remove it.
+
+<Note>More information about the new Wallet Bridge can be found [above](#wallet-bridge).</Note>
+
+### `SignInWithWorldID` Component
+
+The `SignInWithWorldID` and `SignInButton` components have been removed from IDKit. All applications that wish to use Sign In with Worldcoin should do so through our [OIDC Provider](/id/sign-in).
```
[the remark this answers is not in the document]
Comment (0xPenryn) [reply src/pages/reference/v1-migration-guide.mdx]: created a linear issue for this, deprioritizing for now

File: src/components/Navigation.tsx [hunk 5]
```diff
@@ -200,6 +201,7 @@ export const navigation = [
 			{ title: 'OIDC Explainer', href: '/further-reading/oidc' },
 			{ title: 'Protocol Internals', href: '/further-reading/protocol-internals' },
 			{ title: 'Zero-Knowledge Proofs', href: '/further-reading/zero-knowledge-proofs' },
+			{ title: 'World ID Lite', href: '/further-reading/world-id-lite' },
```
Comment (pdtfh) [inline src/components/Navigation.tsx:204]: let's keep it under the main World ID section, at least for a few months

File: src/pages/index.mdx [hunk 6]
```diff
@@ -1,12 +1,21 @@
 <span className="mb-9 mt-4 flex rounded-3xl bg-gray-900 bg-[url('/images/docs/announcement-texture.png')] bg-repeat px-8 py-1 text-white">
 	<span className="grow">
-		<span className="block text-lg font-bold">World ID SDK now publicly available</span>
-		The SDK is now available for public use. Read the announcement and start building now.
+		<span className="block text-lg font-bold">World ID 2.0 is live!</span>
```
Comment (pdtfh) [inline src/pages/index.mdx:3]:
```suggestion
		<span className="block text-lg font-bold">Announcing World ID 2.0!</span>
```

File: src/pages/further-reading/world-id-lite.mdx [hunk 7]
```diff
@@ -0,0 +1,15 @@
+# World ID Lite
+
+World ID Lite is replacing the previous `phone` credential in World ID. World ID Lite relies on the user's identity wallet, such as World App, to verify a user's unique device.
+
+The premise of World ID Lite is users having a unique device. This is more scalable, user friendly, and privacy-preserving than the previous `phone` credential, which was issued when a user verified their phone number in World App.
+
+<Note>Read about configuring IDKit for World ID Lite in the [IDKit Reference](/reference/idkit/).</Note>
+
+## Benefits
```
Comment (pdtfh) [inline src/pages/further-reading/world-id-lite.mdx:9]: We need to add a section above this, with suggestions on when to use World ID Lite and the trade-offs versus using the Orb.

File: src/pages/further-reading/world-id-lite.mdx [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+
+World ID Lite has a number of benefits over the previous `phone` credential:
+
+- **Automatic Issuance:** World ID Lite is automatically issued to all World App users with a unique device. No action needs to be taken by users to receive this credential.
+- **Privacy:** Users can now receive a World ID without providing any information at all. Although verifying a phone number in World App has always been optional, users could previously only use World ID after they verified their phone number or verified at an Orb.
+- **Credential Strength:** World ID Lite is a stronger credential than the previous `phone` credential. It is more difficult and more expensive for an adversary to obtain a new unique device than a new phone number.
```
Comment (pdtfh) [inline src/pages/further-reading/world-id-lite.mdx:15]:
```suggestion
```

**Review by pdtfh — APPROVED** [review]
merging to avoid conflicts, let's address outstanding comments in a separate PR

**Pull request merged**